public_html/wp-content/plugins/pattern-directory/includes/pattern-post-type.php

```diff
@@ -716,6 +728,28 @@ function filter_patterns_rest_query( $args, $request ) {
 		}
 	}
 
+	// If `curation` is passed and either `core` or `community`, we should
+	// filter the result. If `curation=all`, no filtering is needed.
+	if ( isset( $request['curation'] ) ) {
+		if ( 'core' === $request['curation'] ) {
+			// Patterns with the core keyword.
+			$args['tax_query']['core_keyword'] = array(
+				'taxonomy' => 'wporg-pattern-keyword',
+				'field'    => 'slug',
+				'terms'    => 'core',
+				'operator' => 'IN',
```

Comment:
This looks like the only difference between the queries? If so, is this worth simplifying?

Reply:
The operator is the only difference, but I like that this keeps the full sub-query together.

Reply:
Yep. 👍